strings/sillytips.txt

```diff
@@ -45,3 +45,12 @@ Help.
 Maints.
 BZ stops or slows down Lings chem regeneration drastically, make sure to BZ flood the station when lings are confirmed!
 Admins always regret meme options in their polls.
+ERP stands for Emergent Role Play.
+Remember if you see a bug report it on Github.
+Have good ideas for the game? Well make a suggestion on Github, we have templates just for it!
+Role Play is fun. Try it sometimes.
+This is a MRP Server, no really.
+Clean bot has more to say after being emaged. Try it sometimes with your limited use Emags... It will be worth it!
+Bar sign can be emaged.
```

Comment:
Pretty sure it would be spelled "emagged" but it's a made up word so who knows.

Reply:
e like the letter
mag like a /mag/net right?
so it should be 1 g?

Reply:
pretty sure the past tense of every verb that ends in a G has two G's. Lugged, sagged, hugged, gagged, etc.